### PR TITLE
AnagramHomework

### PR DESCRIPTION
**Pull request by theprostonikol**

### CODE REVIEW

File: src/anagram/AnagramChecker.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import java.util.Arrays;
+public class AnagramChecker {
+    public static boolean isAnagram(String s, String t) {
+
+        if (s.length() != t.length()) {
+            return false;
+        }
+
+
+        char[] sArray = s.toCharArray();
```
Comment (shaitanaleksander) [inline src/anagram/AnagramChecker.java:11]: bad var name

File: src/anagram/AnagramChecker.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        Arrays.sort(sArray);
+        char[] tArray = t.toCharArray();
```
Comment (shaitanaleksander) [inline src/anagram/AnagramChecker.java:13]: same

File: src/anagram/AnagramChecker.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    public static void main(String[] args) {
+        String s1 = "anagram";
+        String t1 = "nagaram";
+        System.out.println(isAnagram(s1, t1)); // true
+
+        String s2 = "rat";
+        String t2 = "car";
+        System.out.println(isAnagram(s2, t2)); // false
+    }
+}
```
Comment (shaitanaleksander) [inline src/anagram/AnagramChecker.java:34]: you need to work on variable mane names like s1 t1 ,.... is a bad practice

**Review by shaitanaleksander — APPROVED** [review]

File: src/anagram/AnagramChecker.java (hunk 1 above)
Comment (theprostonikol) [reply src/anagram/AnagramChecker.java:11]: strToCharArr will be good?

**Pull request merged**